[Async Clipboard article] Fix iframe permissions policy attributes, add explanation (#4101)
GoogleChrome/web.dev: src/site/content/en/blog/async-clipboard/index.md

}, 2000);
```

## Permissions policy integration

> In order to use the API in iframes, you need to enable it by making use of the

Reviewer: ```suggestion
To use the API in iframes, you need to enable it with
```

[Permissions Policy](https://w3c.github.io/webappsec-permissions-policy/),

> which defines a mechanism that allows developers to selectively enable and
> disable various browser features and APIs. Concretely, you need to pass either

Reviewer: ```suggestion
which defines a mechanism that allows for selectively enabling and
disabling various browser features and APIs. Concretely, you need to pass either
```
